### PR TITLE
[BPK-3684] Trigger RN build after release

### PR DESCRIPTION
**Pull request by tiagoengel**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/master/CONTRIBUTING.md)

This is a build I triggered locally using the script https://travis-ci.org/github/Skyscanner/backpack-react-native/jobs/670047330#L1153 

NOTE that it should fail as the lates version of backpack-android does not work with backpack-react-native 

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: scripts/trigger-rn-build.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}"
+
+echo "Triggering a backpack-react-native build using backpack-android $backpack_version"
+echo $body
+
+curl -s --fail --show-error -X POST \
+  -H "Content-Type: application/json" \
+  -H "Accept: application/json" \
+  -H "Travis-API-Version: 3" \
+  -H "Authorization: token $TRAVIS_API_KEY" \
```
Comment (georgegillams) [inline scripts/trigger-rn-build.sh:31]: Is this backpackbot's key?

**Comment by tiagoengel** [comment]
This is done by the RN CI now so we only need to trigger a new build and if something is wrong the build will fail and emails us :)

File: .travis.yml [hunk 2]
```diff
@@ -72,7 +72,7 @@ script:
 
   - if [ "$TRAVIS_BRANCH" == 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then openssl aes-256-cbc -K $encrypted_8363dc54ada0_key -iv $encrypted_8363dc54ada0_iv -in backpack.android.demo.key.enc -out backpack.android.demo.key -d ; fi
   - if [ "$TRAVIS_BRANCH" == 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then ./gradlew app:assembleOssRelease && curl -F ipa=@app/build/outputs/apk/oss/release/app-oss-release.apk -F status=2 -F strategy=replace -H X-HockeyAppToken:$HOCKEYAPP_TOKEN https://rink.hockeyapp.net/api/2/apps/upload ; fi
-  - if [ "$TRAVIS_BRANCH" == 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then ./scripts/compile-bridges.sh ; fi
+  - if [ "$TRAVIS_BRANCH" == 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then ./scripts/trigger-rn-build.sh ; fi
```
Comment (georgegillams) [inline .travis.yml:75]: we already do something similar on line 81. Could we align the two API calls to make them use the same script, token etc

File: scripts/trigger-rn-build.sh (hunk 1 above)
Comment (tiagoengel) [reply scripts/trigger-rn-build.sh:31]: not sure, I didn't add this, was already available.

**Pull request merged**